docs/calculations.mdx

```diff
@@ -145,7 +147,9 @@ Native product carousel
 | `device_type`                              | See [device types](#device-types)                                                                           |
 | `network_type`                             | See [network types](#network-types)                                                                         |
 | `property`                                 | See [property](#property)                                                                                   |
-| `creative_ad_format`                       | Either a basic or vendor-provided ad format                                                                 |
+| `identifier`                               | The identifying information, including name, channel, and device type                                       |
+| `vendor`                                   | The name of the serving vendor                                                                              |
+| `ad_format`                                | Either a basic or vendor-provided ad format                                                                 |
```

Comment:
I'm not sure why we have both `identifier` and `ad_format` here? 

The identifier would also just be an identifying information. I think `including name, channel, and device type` might be confusing what the user is meant to provide